src/etc/systemd/system/getty@.service.d/50-oem-autologin.conf

```
@@ -0,0 +1,3 @@
+[Service]
+ExecStart=
+ExecStart=-/sbin/agetty -o '-p -f -- \\\\u' --autologin oem --noclear %I \$TERM
```

```suggestion
ExecStart=-/sbin/agetty -o '-p -f -- \\u' --autologin oem --noclear %I $TERM
```
`\\` is to escape in bash.